extension/WebJobs.Extensions.RabbitMQ.Tests/RabbitMQClientBuilderTests.cs

```diff
@@ -23,8 +23,8 @@ public void Opens_Connection()
             mockServiceFactory.Setup(m => m.CreateService(It.IsAny<string>(), false)).Returns(new Mock<IRabbitMQService>().Object);
             RabbitMQAttribute attr = GetTestAttribute();
 
-            RabbitMQClientBuilder clientBuilder = new RabbitMQClientBuilder(config, options);
-            var model = clientBuilder.Convert(attr);
+            var clientBuilder = new RabbitMQClientBuilder(config, options);
+            IModel model = clientBuilder.Convert(attr);
```

Comment:
Use `var` when the type is evident e.g., when we are new-ing a class object. Use actual type otherwise e.g., for storing function return values. The styles are enforced to make choice of var vs no-var consistent across all files.